content/automate/ManageIQ/Transformation/TransformationHosts/Common.class/__methods__/utils.rb

```diff
@@ -43,9 +43,9 @@ def self.get_runners_count_by_ems(ems, factory_config)
 
             def self.get_transformation_host(task, factory_config, handle = $evm)
               ems = handle.vmdb(:ext_management_system).find_by(:id => task.get_option(:destination_ems_id))
-              ems_max_runners = ems.custom_get('Max Transformation Runners').to_i || factory_config['ems_max_runners'] || 1
+              ems_max_runners = ems.custom_get('Max Transformation Runners') || factory_config['ems_max_runners'] || 10
```

Comment:
I would expect that `ems_max_runners` is an integer and we should not have to call `to_i` on line 48 when you are going to evaluate it.  Also `10` should be a class constant `DEFAULT_EMS_MAX_RUNNERS`.

```RUBY
        module Common
          class Utils
            DEFAULT_EMS_MAX_RUNNERS = 10

            def initialize(handle = $evm)
```
You could do something like this:
`ems_max_runners = (ems.custom_get('Max Transformation Runners') || factory_config['ems_max_runners'] || DEFAULT_EMS_MAX_RUNNERS).to_i`

A better refactoring would be to move this logic into a method where you could potentially log where the value was coming from if you felt that was important.

```ruby
def ems_max_runners
  if ems.custom_get('Max Transformation Runners')
    <log message>
    ems.custom_get('Max Transformation Runners').to_i
  elsif factory_config['ems_max_runners']
    <log message>
    factory_config['ems_max_runners'].to_i
  else
    <log message>
    DEFAULT_EMS_MAX_RUNNERS
end
```

Reply:
`ems_max_runners` is not an integer when it is a custom attributes, because custom attributes are stored as strings.
Anyway, it's a good idea to create a method, as the message will help debugging.